### DISCUSSION TITLE
Can we add --web flag to route:list command

### DISCUSSION THREAD

**Discussion by ArmaghanKhan47** [laravel/framework#39746] [Ideas] [answered]
When --web flag written with route:list command, it should open a web page displaying all the routes and show routes in there more elegantly.

Can it be possible to open a web page from artisan command?

**Comment by s-patompong** [comment, marked as answer]
Yeah, this is a good idea. Long routes would be easier to read on the browser (with the horizontal scroll).

**Comment by s-patompong** [comment]
Using the idea from this conversation, I created this package https://github.com/s-patompong/laravel-routes-html that will add a `/route-list` (customizable) route to the Laravel application and it allows us to look at our route list on the browser.

I have no idea how to integrate `--web` to Laravel internal so that's the most I can do :)
